src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        class="search_form"
+        action="form"
+        method="post"
+        data-qa="big"
+      >
+        <label class="form_input">
+          <img
+            src="images/Search.svg"
+            alt="search"
+            class="search_icon"
+          >
+          <input
+            type="text"
+            data-qa="keypress"
+            placeholder="Try &quot;Los Angeles&quot;"
+          >
+        </label>
+      </form>
+      <form
+        class="search_form"
+        action="form"
+        method="post"
+        data-qa="small"
+      >
+        <label class="form_input small_input">
+          <img
+            src="images/Search.svg"
+            alt="search"
+            class="search_icon small_icon"
```

Comment:
```suggestion
            class="search_icon search_icon-small"
```
not critical, but better in this way